src/reactable/table.jsx

```diff
@@ -133,6 +133,35 @@ export class Table extends React.Component {
         this.tfoot = tfoot;
 
         this.initializeSorts(props);
+        this.initializeFilters(props);
+    }
+
+    initializeFilters() {
+        this._filterable = {};
+        // Transform filterable properties into a more friendly list
+        for (let column of this.props.filterable) {
```

Comment:
it's this that's killing the tests. quite embarrassingly, i still can't figure it out